### PR TITLE
    PATRON2020-69 Protokół komunikacji Gateway-podstacje

### PR DESCRIPTION
**Pull request by SzymonFed**

### CODE REVIEW

**Comment by SzymonFed** [comment]
Sample jsons with informations not used by backend based on https://tracker.intive.com/jira/browse/PATRON2020-69 task

File: communication_protocol/lightSetup.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "id" : 1,
+    "type" : "lightColor",
+    "value" : "0x57EB56"
```
Comment (Siwy160) [inline communication_protocol/lightSetup.json:4]: Why that way? We already have json for lights in API documentation.
https://intive.github.io/patronage20-concept/#put-/light

And why you switched light value format to rgb?

Comment (kuba8686) [reply communication_protocol/lightSetup.json:4]: JSON with color information for LED lights was was included in a comment to the ticket https://tracker.intive.com/jira/browse/PATRON2020-69. But as You said we have it at api doc, so not needed here anymore, as an extra one.

**Comment by SzymonFed** [comment]
Since we don't have keyboard and RFID, an hvac servos are already done i think we can close this PR

**Pull request closed without merging**